### ISSUE TITLE
playback speed usability options

### ISSUE THREAD

**Issue by gruianovac** [podverse/podverse-rn#1733]
currently only can increase speed by touching. no way to reset or go back.  please add button for reset or reverse

**Comment by mitchdowney** [comment]
@gruianovac I agree it's tedious to have to keep tapping the speed button to get to the new speed you're looking for. There is not a lot of room on the player for extra buttons though...if we add advanced speed features, I'm thinking maybe we make it so pressing and holding the speed button pops up a speed menu? From there you could tap the speed you're looking for, instead of cycling through them all.

**Comment by staticdev** [comment]
@mitchdowney The only thing I miss moving from AntennaPod to PodVerse is the ability to choose any speed and select as preset for use later. For me the usability of speed selection in AntennaPod is much better and could be used as inspiration for implementation since is also open-source.

**Comment by mitchdowney** [comment]
@staticdev could you help outline how you'd like it to work more exactly? I can definitely research AntennaPod's implementation someday, but I'm working a huge rewrite of our infrastructure right, and if you're able to provide those details I would really appreciate it.

**Comment by staticdev** [comment]
@mitchdowney sure, so how it works:

Antennapod https://github.com/AntennaPod/AntennaPod is in Java, so implementation should not be super userful.
When you click current speed you will see a selector of presets or a bar where you can choose from 0.5 to 4x:

![antennapod1](https://github.com/podverse/podverse-rn/assets/1446633/505a0969-0593-4936-bd87-ae6f5aae3224)

When you click the plus button, it will add the new selected speed to the list of presets:

![antennapod2](https://github.com/podverse/podverse-rn/assets/1446633/4f48f401-5287-4111-ae89-33933456dea2)

An honorable mention is NewPipe (Youtube player) also open-source in Java/Kotlin - https://github.com/TeamNewPipe/NewPipe with similar behavior - but you can select from 0.1x to 3x.

![newpipe](https://github.com/podverse/podverse-rn/assets/1446633/2dbd3fd2-4b11-4528-b23f-2d008bc99ead)

 If you need more details feel free to ask.

**Comment by mitchdowney** [comment]
Thanks @staticdev that helps a lot. It shouldn't be too difficult to use a similar feature design as AntennaPod. I'm not a frequent playback speed adjuster though, so it's not totally intuitive to me that a multi-tap playback speed process would be preferable. For me, I kind of like just a single tap/s to scroll through playback speeds as needed. In any case, a feature like this should be feasible, I just wonder if this is a design that "more advanced" listeners prefer, or more listeners in general prefer. In other words I wonder if it should be a setting people enable or just the behavior for all listeners 🤔

**Comment by staticdev** [comment]
It is basically 1 tap - you see a slide and choose. Or 1 tap and another tap for a preset.
Playback speed for me is very standard feature, I have also friends and family that hear podcasts, it is common to hear they commenting about increasing speed mostly.

I understand your concert, not really sure if you have a mechanism to receive feedback.. on F-Droid unfortunately it is not possible to comment or evaluate apps. But I really believe the UX for speed is better like this.

**Comment by barrelful** [comment]
I completely agree with @staticdev, having a sliding bar for choosing the speeds is very handy, I am also user of NewPipe and I love using this feature.

**Comment by catarse-natural** [comment]
@mitchdowney another fan of speed bar here. Would be great if we can have it!

**Comment by mitchdowney** [comment]
We are archiving all Github issues and PRs that pertain to the previous generation Podverse. The Podverse dev team is currently 100% focused on the next generation app, and will be only working on next gen Github issues going forward.

If you would like to see a work-in-progress demo of the upcoming Podverse website, please visit https://alpha.podverse.fm. Our tentative release date is Spring 2026.

If you're a developer who may be interested in contributing to Podverse Next-Gen, please see our [Local Dev Setup documentation](https://github.com/podverse/podverse-ops/blob/v5-develop/dev/local-dev-setup.md). We can also be reached through [Matrix or Discord](https://podverse.fm/contact).

I'm sorry for the abrupt announcement, but thank you for your bug reports and feature requests.
